src/inverseRobot.js

```diff
@@ -2,12 +2,28 @@
 
 /*
  * @param {object} robot
- *
- * @return {object}
+*
+* @return {object}
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  let checkDoubleProper = false;
+
+  Object.entries(robot).forEach((key) => {
+    if (newRobot.hasOwnProperty(key[1])) {
+      checkDoubleProper = null;
+
+      return;
+    }
+    newRobot[key[1]] = key[0];
+  });
+
+  if (checkDoubleProper === null) {
+    return null;
```

Comment:
The condition 'if (checkDoubleProper === null)' is unnecessary because if the code execution reaches this point, it means that there are no duplicate properties and you can directly return the 'newRobot' object.